platforms-serverless/netlify-github-with-nextjs-caching/prepare.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eu
+
+export PRISMA_TELEMETRY_INFORMATION='ecosystem-tests platforms-serverless netlify-github-with-nextjs-caching build'
+
+rm -fr .git # because caanot currently be done in finally.sh if failure happens
```

Comment:
What is this needed for?

Reply:
When there's a failure and the run is fully retried, there is no way to properly clean it up because `finally.sh` is not called. So the git commands will actually fail. To work around that we always delete the git files. The actual problem might actually be that we don't reset the test folder properly, so maybe we should properly reset before running a test so it can be re-run without keeping the previous state. Something along these lines:

```sh
git reset HEAD --hard && git clean -f -x && <run test command>
```

Reply:
Created a draft issue as a reminder